.github/workflows/ci.yml

```diff
@@ -23,6 +25,18 @@ jobs:
         run: npm install
       - name: Run Tests
         run: npm run ci
+      - name: Run sign benchmark
+        if: ${{matrix.node-version > 12}}
+        run: npm run benchmark:sign
+      - name: Run decode benchmark
+        if: ${{matrix.node-version > 12}}
+        run: npm run benchmark:decode
+      - name: Run verify benchmark
+        if: ${{matrix.node-version > 12}}
+        run: npm run benchmark:verify
+      - name: Run auth0 benchmark
+        if: ${{matrix.node-version > 12}}
+        run: npm run benchmark:auth0
```

Comment:
let's put this in a multiline script instead of duplicating the conditional over and over

Reply:
done 👍